plugins/shared-react/src/utils/date/format-date.ts

```diff
@@ -1,9 +1,29 @@
+/* export function formatDate(date: string | number | Date) {
+  if (date === -1) {
+    return 'N/A';
+  }
+
+  const adjustedDate = typeof date === 'number' ? date * 1000 : date;
+  return new Date(adjustedDate).toLocaleString('en-US', {
+    timeZone: new Intl.DateTimeFormat().resolvedOptions().timeZone,
+    timeStyle: 'short',
+    dateStyle: 'medium',
+  });
+} */
+
 import { format } from 'date-fns';
 
-export function formatDate(date: string | undefined) {
-  if (!date) {
+/**
+ * Returns the given date as a formated Date.
+ *
+ * @param date - The given date in seconds
+ * @return The date formatted to en-US locale, otherwise return 'N/A'
+ */
+export function formatDate(date: string | number | Date | undefined): string {
```

Comment:
should we take changes in shared-react in separate PR and once it's published then update other plugins consuming it?

Reply:
The releaser can handle this. If we update 2 packages in one single PR where package A depends on package B, it will:

- release new version of package B
- bump version of package B in package A dependencies
- release new version of package A

This was one of the requirements I had on the release toolkit so we can effectively update `-common` packages and its dependants

Reply:
@invincibleJai Can this thread be resolved?